tools/run-dev

```diff
@@ -377,7 +380,7 @@ async def serve() -> None:
     else:
         children.append(start_webpack_watcher())
 
-    if options.help_center:
+    if options.help_center and not options.skip_help_center_build:
```

Comment:
For this commit, the commit message should explain what this option is useful for. E.g. add a new paragraph "It takes about 30 seconds on my system to build all the help center articles, so it's useful to skip that when not iterating on the conversion process."

I'm not sure this is the right design, though; should `run-dev` be responsible for building it at all? Since needing to build it is a temporary step only used for testing, I think probably the right plan is to not have `run-dev` support building it, and have the instructions for testing the beta help center involve running the build step manually as needed.